appium/webdriver/appium_connection.py

```diff
@@ -25,17 +25,35 @@
 
 
 class AppiumConnection(RemoteConnection):
+    def __init__(
+        self,
+        remote_server_addr: str,
+        keep_alive: bool = False,
+        ignore_proxy: Optional[bool] = False,
+        init_args_for_pool_manager: Union[Dict[str, Any], None] = None,
+    ):
+
+        # Need to call before super().__init__ in order to pass arguments for the pool manager in the super.
+        if init_args_for_pool_manager is None:
```

Comment:
could be simplified to `self._init_args_for_pool_manager = init_args_for_pool_manager or {}`